src/components/Reports/SalaryCalculator/Landing/NewSalaryCalculationLanding/NewSalaryCalculatorLanding.tsx

```diff
@@ -0,0 +1,118 @@
+  paddingBottom: theme.spacing(4),
+}));
+
+export const NewSalaryCalculatorLanding: React.FC = () => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const {
+    loading,
+    staffAccountId,
+    name,
+    self,
+    spouse,
+    isNewStaff,
+    salaryData: { currentGrossSalary },
+    accountBalance,
+  } = useLandingData();
+
+  const { startCalculation, calculation } = useSalaryCalculator();
+  const hasInProgressCalculation =
+    calculation?.status === SalaryRequestStatusEnum.InProgress;
+
+  const assignmentCategory = self?.staffInfo.assignmentCategory;
+  const isFullTime =
+    assignmentCategory === AssignmentCategoryEnum.FullTimeRegular ||
+    assignmentCategory === AssignmentCategoryEnum.FullTimeTemporary;
+
+  const isSelfSupportedRmo =
+    self?.staffInfo.peopleGroupSupportType ===
+    PeopleGroupSupportTypeEnum.SupportedRmo;
+  const isSpouseSupportedRmo =
+    spouse?.staffInfo.peopleGroupSupportType ===
+    PeopleGroupSupportTypeEnum.SupportedRmo;
+  const isSupportedRmo = isSelfSupportedRmo || isSpouseSupportedRmo;
+
+  if (loading) {
+    return <Loading loading />;
+  }
+
+  if (!staffAccountId) {
+    return <NoStaffAccount />;
+  }
+
+  return (
+    <StyledContainer>
+      <Box>
+        <Box sx={{ marginBottom: theme.spacing(3) }}>
+          <Typography variant="h4" component="h1" gutterBottom>
+            {t('Salary Calculation Form')}
+          </Typography>
+          <Typography variant="body1" paragraph>
+            {t(
+              'Using this form will enable you to make adjustments to your salary. Your current salary information is displayed below. To begin a new salary calculation, click on "Calculate New Salary." Your Maximum Allowable Salary includes your salary, taxes, SECA, and 403(b).',
+            )}
+          </Typography>
+          <Typography variant="body1" paragraph>
+            {t(
+              'Your 403(b) contribution election percentage(s) are shown below. If you would like to make changes you may do so by logging into',
+            )}{' '}
+            <Link
+              href="https://accounts.principal.com/app/bookmark/0oadm2qe1orihoKba5d7/login"
+              target="_blank"
+              rel="noopener noreferrer"
+              style={{ color: theme.palette.primary.main }}
+            >
+              {t('your Principal account')}
+            </Link>
+            .{' '}
+            {t(
+              'Please wait to complete your Salary Calculation Form until those changes are reflected here.',
+            )}
+          </Typography>
+        </Box>
+        <NameDisplay
+          names={name}
+          personNumbers={staffAccountId ?? ''}
+          showContent
+          titleOne={t('Current Gross Salary')}
+          titleTwo={t('Account Balance')}
+          amountOne={currentGrossSalary}
+          amountTwo={accountBalance}
+        />
+        <SalaryInformationCard />
+        {(isFullTime || isSupportedRmo) && !isNewStaff && (
```

Comment:
I don't think that full-time employees who aren't RMO can change their salaries.

Reply:
It seems like you would want the `!isNewStaff` check too: `isFullTime && isSupportedRmo && !isNewStaff`. Where did you find "State: Completed Full-time && Single || SOSA"? Can you link to it? I have a hunch that that condition is for whether or not to show the spouse's information (in this case, don't show the spouse info because they are single or SOSA).